icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -135,12 +135,14 @@ export ICU4C_DIR=$HOME/icu-myfork/icu4c
 export ICU4J_ROOT=$HOME/icu-myfork/icu4j
 export TOOLS_ROOT=$HOME/icu-myfork/tools
 
-# 2. Build and install the CLDR jar
+# 2. Build and install the CLDR jar, as well as installing some CLDR
+# testData files needed by ICU4C and ICU4J tests.
 
-cd $TOOLS_ROOT/cldr/lib
-./install-cldr-jars.sh "$CLDR_DIR"
+cd $TOOLS_ROOT/cldr
+ant
```

Comment:
So this won't run the actual CLDR-to-ICU tools at this point.
Since the default action is not "do everything" I'd be tempted to give an explicit target here (e.g. "ant prepare-cldr" or "cldr-ant init"). Default actions are only good when they're the only thing you ever normally need.

Reply:
If the default action isn't recommended, should it be disabled somehow? (Is that possible?)

Is "ant install" appropriate? I've gone with that, removing one step of "default" indirection. But perhaps, considering "clean" and "install"'s usual meaning, maybe you don't want this as "install"?

```
<project name="CLDR" **default="install"** basedir=".">
[...]
    <target name="install" depends="clean, copy-cldr-testdata, **install-cldr-libs**"/>
[...]
```

Reply:
How do you feel about invoking the script, vs invoking the install-cldr-libs ant target (one more indirection step, but a step towards consolidation)?

Reply:
If the Ant script works as well (or better) then it seems like a step forward. Using "install"  feels fine for now, that's more a question for Peter et al.

As for default tasks, if there's no way to disable one, we can make a default action which just does a "help page".